src/AnnouncementBubble/DeletedAnnouncementBubble.tsx

```diff
@@ -10,7 +10,8 @@ function cssClass(element: string) {
 }
 
 export interface Props {
-  theme: "deleted";
+  bubbleType: "deleted";
+  theme?: "launchpad" | "familyPortal";
```

Comment:
Quick thought! Rather than calling the theme `"launchpad"` (since that's more of an internal name than a product name), maybe we can just call it `"default"`. (I also considered `"studentTeacherPortal"` for a sec, but there's a world where we differentiate the student and teacher Portals, so that's not the most future proof)

Reply:
Also, it might make sense to define this theme in a shared type in [the src/utils/ directory](https://github.com/Clever/components/tree/master/src/utils), e.g.

```ts
type MessagingTheme = "default" | "familyPortal";
```

Reply:
Would this be in its own file, or in the types.ts file (for example)

Reply:
I think it could probably live in its own file! Maybe `messaging.ts`, so that all our team's constants are grouped together for an eventual move to a standalone Messaging components library